lib/resty/etcd/v3.lua

```diff
@@ -291,10 +292,11 @@ local function set(self, key, val, attr)
 end
 
 local function get(self, key, attr)
+
     -- verify key
-    key = utils.normalize(key)
-    if not key or key == '/' then
-        return nil, "key invalid"
+    local _, err = utils.verify_key(key)
```

Comment:
we need to add test case for this changing

Reply:
i have added `etcdIns:get` and `etcdIns:set` empty key's test case